submissions/mxnvkv/DOM task/css/styles.css

```diff
@@ -0,0 +1,245 @@
+@import url('https://fonts.googleapis.com/css?family=Work+Sans');
+/* font-family: 'Work Sans', sans-serif; */
+@import url('https://fonts.googleapis.com/css?family=Lato');
+/* font-family: 'Lato', sans-serif; */
+
+/*
+--------------------------------------------------------------------
+********************************************************************
+
+                                Navigation
+
+********************************************************************
+--------------------------------------------------------------------
+*/
+
+nav {
+    position: fixed;
+    z-index: 1;
+    top: 0;
+    left: 0;
+
```

Comment:
Please use one-style for your code. IMO you can delete this empty line. Also check yourself in following code.